src/app/pages/courses-page/course-item/course-item.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="row course-item-card">
+  <div class="col-10">
+    <div class="row card-body">
+      <div class="col-8">
+        <h5 class="card-title">{{ course?.title }}</h5>
+      </div>
+      <div class="col-4 edit-delete-button">
+        <span>{{ course?.duration }} min</span>
+        <span>{{ course?.creationDate | date }}</span>
+      </div>
```

Comment:
1.create separate folder for components, like appcomponent place all files inside appcomponent .
2. Inside modules create folder of component and add all components in it.
3. It would be breadCumbsModules.
4. it would be footerComponent.

Reply:
refactored